Dockerfile

```diff
@@ -15,52 +15,75 @@
 #
 
 FROM ubuntu:22.04 as tools-env
-ARG IMAGE_KERNEL_VERSION=v0.19.1
-ARG LINUX_VERSION=6.5.9-ctsi-1
+ARG IMAGE_KERNEL_VERSION=v0.20.0-output-unification-test1
+ARG LINUX_VERSION=6.5.13-ctsi-1-output-unification-test1
 ARG LINUX_HEADERS_URLPATH=https://github.com/cartesi/image-kernel/releases/download/${IMAGE_KERNEL_VERSION}/linux-libc-dev-riscv64-cross-${LINUX_VERSION}-${IMAGE_KERNEL_VERSION}.deb
 ARG BUILD_BASE=/opt/cartesi
 
 # Install dependencies
 # ------------------------------------------------------------------------------
 ENV LINUX_HEADERS_FILEPATH=/tmp/linux-libc-dev-riscv64-cross-${LINUX_VERSION}-${IMAGE_KERNEL_VERSION}.deb
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install -y --no-install-recommends \
-        build-essential \
+RUN <<EOF
```

Comment:
We should avoid this idiom. It builds images instead of failing when commands error out, a has a different behavior than what is expected.

Check the example below:
```
   FROM busybox                                                                                                                                                                                                   
   RUN <<EOF                                                                                                                                                                                                      
   echo ok                                                                                                                                                                                                        
   error # this should fail                                                                                                                                                                                       
   echo ok                                                                                                                                                                                                        
   EOF                                                                                                                                                                                                            
```

We have:
`#5 0.098 /bin/sh: error: not found`

But it still builds the image.

(Complete log of example)
```
$ docker build --progress=plain -f Dockerfile  .
#0 building with "default" instance using docker driver

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 105B done
#1 DONE 0.0s

#2 [internal] load .dockerignore
#2 transferring context: 2B done
#2 DONE 0.0s

#3 [internal] load metadata for docker.io/library/busybox:latest
#3 DONE 0.5s

#4 [1/2] FROM docker.io/library/busybox@sha256:c3839dd800b9eb7603340509769c43e146a74c63dca3045a8e7dc8ee07e53966
#4 CACHED

#5 [2/2] RUN <<EOF (echo ok...)
#5 0.098 ok
#5 0.098 ok
#5 0.098 /bin/sh: error: not found
#5 DONE 0.1s

#6 exporting to image
#6 exporting layers done
#6 writing image sha256:43dad32f9eb2ed87094e40167445150872928dbc15716bcbe7397f72b4077f9c done
#6 DONE 0.0s
```

Reply:
`set -e` avoid the issue. I'm ok with it.